src/by/naumenka/lesson21/Person.java

```diff
@@ -0,0 +1,41 @@
+package by.naumenka.lesson21;
+
+public class Person {
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private int age;
+
+    public Person(String firstName, String lastName, String sex, int age) {
```

Comment:
In general, it's better to avoid constructors with parameters and initialize everithing in the object through setters